projects/plugins/backup/src/js/components/Admin.js

```diff
@@ -51,6 +54,14 @@ const Admin = () => {
 				setCapabilitiesError( 'Failed to fetch site capabilities' );
 			}
 		);
+		apiFetch( { path: '/jetpack/v4/backup-promoted-product-info' } ).then( res => {
```

Comment:
If this call fails, it results in a blank page. I'm not sure how to gracefully handle this case (because we have no design) or if it's worth it. Maybe grey out the price at $0 but leave the button clickable?

<img width="1724" alt="Screenshot 2021-12-22 at 19 38 49" src="https://user-images.githubusercontent.com/298685/147140043-aa87d810-d67a-4647-9ce3-567b049911b5.png">
t